solvation_analysis/analysis_library.py

```diff
@@ -104,6 +104,14 @@ def shell_percent(self, shell_dict):
         -------
         float
             the percentage of shells
+
+        Examples
```

Comment:
These are not rendering properly as they are not in a code block.

You need to use the 
```
.. code-block:: python
    import MDAnalysis as mda
    import MDAnalysis.analysis.msd as msd
    from MDAnalysis.tests.datafiles import RANDOM_WALK, RANDOM_WALK_TOPO
```

kind of syntax to get proper highlighting.